examples/php-features-test/test-all.sh

```
@@ -0,0 +1,262 @@
+                # Check that only client files are generated
+                if find "$output_dir" -name "*ServiceClient.php" | grep -q .; then
+                    print_success "Found client files"
+                fi
+                if ! find "$output_dir" -name "*ServiceInterface.php" | grep -q .; then
+                    print_success "No server interfaces (client-only mode)"
+                fi
+                ;;
+        esac
+        
+        # Check namespace in generated files
+        if [ -f "$output_dir"/*.php ] || find "$output_dir" -name "*.php" | head -1 > /dev/null; then
```

Shellcheck (0.10.0)

[error] -f doesn't work with globs. Use a for loop.

(SC2144)